designsafe/static/styles/ng-designsafe.css

```
@@ -29,7 +29,7 @@
     outline: 0;
 }
 
-.ds-data-browser-wrapper {
+.ds-data-browser-wraper {
```

Shouldn't this be left as `.ds-data-browser-wrapper` (with two 'p's in 'wrapper')?
```suggestion
.ds-data-browser-wrapper {
```